src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days >= 7) {
+    return sum - 50;
```

Comment:
I'd like to add variables for `discount,` that will be reusable in the future) but up for you)

Reply:
yep, this way is better. thank you!